cmd/skopeo/signing.go

```diff
@@ -8,28 +8,25 @@ import (
 	"io/ioutil"
 
 	"github.com/containers/image/v5/signature"
-	"github.com/urfave/cli"
+	"github.com/spf13/cobra"
 )
 
 type standaloneSignOptions struct {
 	output string // Output file path
 }
 
-func standaloneSignCmd() cli.Command {
+func standaloneSignCmd() *cobra.Command {
 	opts := standaloneSignOptions{}
-	return cli.Command{
-		Name:      "standalone-sign",
-		Usage:     "Create a signature using local files",
-		ArgsUsage: "MANIFEST DOCKER-REFERENCE KEY-FINGERPRINT",
-		Action:    commandAction(opts.run),
-		Flags: []cli.Flag{
-			cli.StringFlag{
-				Name:        "output, o",
-				Usage:       "output the signature to `SIGNATURE`",
-				Destination: &opts.output,
-			},
-		},
-	}
+	cmd := &cobra.Command{
+		Use:     "standalone-sign [command options] MANIFEST DOCKER-REFERENCE KEY-FINGERPRINT",
+		Short:   "Create a signature using local files",
+		RunE:    commandAction(opts.run),
+		Example: `skopeo standalone-sign busybox-manifest.json registry.example.com/example/busybox `,
```

Comment:
This example is missing the `KEY-FINGERPRINT`; but on balance I’d probably prefer not giving an example, `standalone-sign` should not be encouraged.